src/components/sidebar/right-sidebar/datepicker/Handle.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from "react";
+import { SliderItem } from "react-compound-slider";
+
+
+const { Component, Fragment } = React;
+
+interface HandleProps {
+    key: string;
+    handle: SliderItem;
+    isActive: Boolean;
+    disabled?: Boolean;
+    domain: number[];
+    getHandleProps: (id: string, config: object) => object;
+}
+
+const activeHandleStyle: React.CSSProperties = {
```

Comment:
Nit: To be consistent with how we style elsewhere, we define these styles in .css files. If I'm not mistaken, divs can have both a className and style property, so we can do something like:

```
<div
  className="activeHandleStyle"
  style={{
    left: `${percent}%`,
  }}
>
```

Reply:
+1